docs/compute-vm-quickstart.md

```diff
@@ -0,0 +1,136 @@
+# Quickstart for Using the Oracle Toolkit for Google Cloud on Compute Engine VMs
+
+This document serves as a quickstart guide and simple reference for using the Oracle Toolkit for Google Cloud on Compute Engine Virtual Machines. It is an abridged version of the more comprehensive [Oracle Toolkit for Google Cloud - Compute Engine VM User Guide](compute-vm-user-guide.md).
+
+## Prerequisite Assumptions
+
+This document assumes that you have:
+
+1. Provisioned your Ansible Control Node, installed Ansible and JMESpath on it, and downloaded the toolkit.
+1. Setup Google Cloud foundational components such as IAM, networking, Google Cloud Storage buckets with the required media staged, and security aspects.
+1. Have either Terraform or the Google Cloud CLI and specifically the **gcloud** utility installed.
+
+If you need additional details to setup any of these prerequisites, refer to Google documentation such as [Google Cloud quickstarts and tutorials](https://cloud.google.com/docs/tutorials) and specifically [Install the gcloud CLI](https://cloud.google.com/sdk/docs/install). And the more detailed [Oracle Toolkit for Google Cloud - Compute Engine VM User Guide](compute-vm-user-guide.md).
+
+## Deploying using the Google Cloud CLI
+
+### Set Supporting Variables
+
+Before beginning, infrastructure locality and networking aspects must be defined and captured (i.e. into shell environment variables for convenience):
+
+```bash
+PROJECT_ID="PROJECT_ID"
+REGION_ID="REGION"
+ZONE_ID="ZONE"
+NETWORK_ID="NETWORK"
+SUBNET_ID="SUBNET"
+NETWORK_TAGS="NETWORK_TAGS"
+
+gcloud config set project ${PROJECT_ID}
+```
+
+Add additional variables for instance specific characteristics such as the VM shape, name, and OS project and image family. For example if using the [C4 machine series](https://cloud.google.com/compute/docs/general-purpose-machines#c4_series) and the latest RHEL8 OS image:
+
+```bash
+MACHINE_TYPE="c4-standard-4"
+IMAGE_PROJECT="rhel-cloud"
+IMAGE_FAMILY="rhel-8"
+```
+
+Finally, define variables for the virtual machine name, and the Google Cloud Storage bucket where the required Oracle software is staged:
+
+```bash
+VM_NAME="INSTANCE_NAME"
+BUCKET_NAME="BUCKET_NAME"
+```
+
+Verify that the required software is available in the specified bucket using:
+
+```bash
+bash ./check-swlib.sh --ora-swlib-bucket ${BUCKET_NAME}
+```
+
+Optionally, add the version you wish to install to the above command using the `--ora-version` argument.
+
+### Create the Compute VM and Block Storage Devices
```

Comment:
Just a thought: now that we have Terraform support, should we just ask people to run TF?  It would probably mean a lot fewer steps...

Reply:
Thinking about this, having manual and terraform instructions is probably still good.